src/alpine/3.8/helix/arm64v8/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM arm64v8/alpine:3.8
+
+# Install .NET Core Dependencies for Alpine
+
+RUN apk update && \
+    apk add --no-cache \
+        autoconf \
+        bash \
+        build-base \
+        clang \
+        clang-dev \
+        cmake \
+        coreutils \
+        curl \
+        gcc \
+        gettext-dev \
+        git \
+        icu-dev \
+        iputils \
+        krb5-dev \
+        libtool \
+        libunwind-dev \
+        libffi \
+        libffi-dev \
+        linux-headers \
+        llvm \
+        make \
+        openssl \
+        openssl-dev \
+        paxctl \
+        py-cffi \
+        python \
+        python-dev \
+        py-pip \
+        sudo \
+        tzdata \
+        util-linux-dev \
+        wget \
+        which \        
+        zlib-dev && \
+    apk -X http://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+        userspace-rcu-dev \
```

Comment:
(nit) - alphabetize.

Reply:
I thought I had the order correct but I missed a couple